core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -77,13 +78,23 @@ public MerlinBuilder withAllCallbacks() {
     }
 
     /**
-     * Sets the internal Merlin Logging state. Uses the TAG : Merlin
+     * Sets the internal Merlin logging state to report to a given LogHandle.
      *
-     * @param withLogging by default Logging is disabled. withLogging = true will enable logging, withLogging = false will disable.
+     * @param logHandle that should be used when logging.
      * @return MerlinBuilder.
      */
-    public MerlinBuilder withLogging(boolean withLogging) {
-        MerlinLog.LOGGING = withLogging;
+    public MerlinBuilder withLogging(Logger.LogHandle logHandle) {
```

Comment:
Whilst I agree that it is better to avoid breaking changes this library uses the "Semantic versioning system" which states that 

> MAJOR version when you make incompatible API changes

The next release is going to be a major release because of the introduction of modules for `RxJava` and some other API breaking changes. I don't like deprecating methods that might have some affect or indeed none on the current library, I think we should make the changes and properly document them i.e. by creating excellent migration documentation #107